vms/platformvm/state/state.go

```diff
@@ -1447,7 +1447,11 @@ func (s *state) loadCurrentValidators() error {
 		}
 		tx, _, err := s.GetTx(txID)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed loading validator transaction txID %v, %w", txID, err)
```

Comment:
moved this check up of a few line, to reduce diffs later on.
This error should never happen, but always better failing fast